app/views/notifications/view.njk

```diff
@@ -0,0 +1,40 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "macros/new-line-array-items.njk" import newLineArrayItems %}
+
+{% block breadcrumbs %}
+  {# Back link #}
+  {{
+    govukBackLink({
+      text: 'Go back to summary',
+      href: '/system/licences/' + licenceId + '/summary'
```

Comment:
We can assume that we got here because someone clicked a link in the communications tab.

So, we can alter our usual pattern for back links by having it return the user to the licence's communications tab, and having the text be `Go back to communications`.

> N.B. I'm aware some idiot told you to **Go back to summary** in the ticket. This is why it is perfectly fine to question those in power! 😬😝